Creating-Objects.md

```diff
@@ -31,7 +31,7 @@ The schema would match a dictionary like this:
 {"version": 34}
 ```
 
-This schema describes a dictionary of ints, and the `int` type is an object. We define these objects as classes in [`extensions/objects/models/objects.py`](https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects.py) and components in [`extensions.objects/templates`](https://github.com/oppia/oppia/blob/develop/extensions/objects/templates).
+This schema describes a dictionary with a `version` key whose value is an `int`, and the `int` type is an object. We define these objects as classes in [`extensions/objects/models/objects.py`](https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects.py) and components in [`extensions.objects/templates`](https://github.com/oppia/oppia/blob/develop/extensions/objects/templates).
```

Comment:
Sorry, this is still a bit confusing. You say "whose value is an int, and the int type is an object" -- this makes it unclear whether int is a type, value, or object.

Actually on re-reading this I think you might be confusing two things. Firstly, there are standard python types: int, bool, etc. -- these are not objects defined by us. 

Secondly, there are custom types like Int, UnicodeString, etc. These are types defined by us and is what is contained in objects.py. Note that objects are not necessarily base types, so the statement "We call these base types 'objects'" is wrong -- though they typically have an internal structure that is constructed out of base Python types. For example, the Int object in objects.py has type "int", and more complex objects have more complex types. Objects are invoked when the 'type' in the schema dict is 'custom'.

Does this clarify? Happy to chat and explain more about this if you like.

Reply:
Yes, I think your explanation helped. I took another stab at explaining what objects are.